_posts/2019-05-29-angular-onpush-strategy.md

```diff
@@ -0,0 +1,93 @@
+---
+layout:    post
+title:     "Angular - detekcja zmian strategią onPush"
+date:      2019-08-08 07:00:00 +0100
+published: false
+author:    kgabara
+tags:
+    - angular
+    - frontend
+---
+Każda aplikacja rozwijana odpowiednio długi czas może rozrosnąć się do ogromnych rozmiarów, a konkretniej do sporej liczby komponentów, jeżeli mówimy o aplikacji frontendowej pisanej z wykorzystaniem Angulara. Duża liczba komponentów może spowodować, że w pewnym momencie nasza aplikacja przestanie spełniać oczekiwania odnośnie wydajności. W takim momencie powinniśmy pomyśleć nad możliwościami naprawy tego problemu. Jednym z rozwiązań mogłaby być zmiana domyślnej strategii detekcji zmian.
```

Comment:
> Jednym z rozwiązań mogłaby być zmiana domyślnej strategii detekcji zmian.

Niezbyt mi sie podoba to zdanie. Może coś podobnego zabrzmiałoby lepiej: "W tym artykule pokażemy jedno z możliwych rozwiązań tego problemu - zmiana strategii detekcji zmian"

Reply:
done